lib/gui/modules/image-writer.js

```diff
@@ -124,7 +125,7 @@ imageWriter.service('ImageWriterService', function ($q, $rootScope) {
 
       if (error.code === 'EVALIDATION') {
         analytics.logEvent('Validation error', analyticsData)
-      } else if (error.code === 'EUNPLUGGED') {
+      } else if (BlockWriteStream.isTransientError(error)) {
```

Comment:
This is still not correct. Anything on the writer can throw these errors, and they would be incorrectly caught here. Instead of using a global handler for these generic errors, we should only check them in `fs.read` and `fs.write`. We should limit the scope of the error handler. That was the reasoning behind `EUNPLUGGED`.